src/Stratis.Bitcoin.Features.Consensus/CoinViews/DBreezeCoinView.cs

```diff
@@ -19,6 +20,8 @@ namespace Stratis.Bitcoin.Features.Consensus.CoinViews
     /// </summary>
     public class DBreezeCoinView : ICoinView, IDisposable
     {
+        public const int MaxBlocksToRewind = 1000;
```

Comment:
This doesn't appear to be used anywhere?

Reply:
Remove